pkg/wamr/Makefile

```diff
@@ -47,6 +47,8 @@ else ifeq ($(CPU_ARCH),xtensa)
   WAMR_BUILD_TARGET = XTENSA
 else ifeq ($(CPU_ARCH),rv32)
   WAMR_BUILD_TARGET = RISCV32
+else ifeq ($(CPU_ARCH),riscv_esp32)
```

Comment:
I'm a bit unhappy that there is a special esp32 riscv `CPU_ARCH` now.
Why can't this be `rv32`?

Reply:
> I'm a bit unhappy that there is a special esp32 riscv `CPU_ARCH` now.

I know.

> Why can't this be `rv32`?

We discussed it already in https://github.com/RIOT-OS/RIOT/pull/17841#pullrequestreview-1003344320 and https://github.com/RIOT-OS/RIOT/pull/17841#issuecomment-1163093696. When I ported RIOT to ESP32-C3, I definitely tried to reuse common parts from `cpu/riscv_common` and implement only some specific parts for ESP32-C3. But since using a `cpu/*` directory means all or nothing and some parts in `cpu/riscv_common` are not compatible with ESP-IDF and ESP32-C3 SoC, a specific architecture had to be defined with its own implementation in PR #18260. I am not happy with it either, but there was no other option.

Reply:
https://github.com/RIOT-OS/RIOT/pull/17844/files#r925545350

Reply:
If that's possible it would certainly make the most sense from a consistency point of view.

Reply:
Seems to work for ESP32-C3 with these changes so that we could close this PR. I have to check Kconfig before.